pkg/models/devops/devops.go

```diff
@@ -698,6 +698,7 @@ func jenkinsClient(baseUrl string, req *http.Request) ([]byte, http.Header, erro
 	defer resp.Body.Close()
 
 	if resp.StatusCode >= http.StatusBadRequest {
+		log.Errorf("%+v", string(resBody))
```

Comment:
also return resBody to frontend ?

Reply:
And do you need to log some context information such as the request URL?

Reply:
no，just log resBody. return error mssage to frontend.
already log context information in previous func.

Reply:
At least need to expose http status code to the front end, always 500 is not a good choice

Reply:
yes，will return resp.StatusCode same as Jenkins returns if StatusCode > 200, not always 500.